roles/zuul/tasks/main.yml

```diff
@@ -164,6 +165,53 @@
     - Reload systemd units
     - Restart zuul
 
+- name: Create zuul status lib dir
+  file:
+    path: /opt/source/zuul/etc/status/public_html/lib
+    state: directory
```

Comment:
This feels wrong, we are creating and adding directories to a git repository. IMO all these files should probably go to /var/www or something and patch it up from apache.